ts/models/conversation.ts

```diff
@@ -397,6 +423,23 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     return true;
   }
 
+  public async updateGroupModerators(groupModerators: Array<string>, shouldCommit: boolean) {
+    if (!this.isPublic()) {
+      throw new Error('group moderators are only possible on SOGS');
+    }
+    const existingModerators = uniq(sortBy(this.getGroupModerators()));
+    const newModerators = uniq(sortBy(groupModerators));
+
+    if (_.isEqual(existingModerators, newModerators)) {
```

Comment:
should import isEqual directly.

Reply:
:+1: fixed that one and a few others too